packages/e2e-tests/specs/editor/various/mentions.test.js

```diff
@@ -10,7 +10,9 @@ import {
 	pressKeyTimes,
 } from '@wordpress/e2e-test-utils';
 
-describe( 'autocomplete mentions', () => {
+// Something in WP 5.9 RC has broken the creation of new users so disabling until
+// that is fixed
+describe.skip( 'autocomplete mentions', () => {
```

Comment:
This is passing in https://github.com/WordPress/gutenberg/pull/36987. Might be worth enabling it again 😄